package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/InstanceImporter.cs

```diff
@@ -126,11 +129,23 @@ public static void BuildSceneInstances(PrimMap primMap, SceneImportOptions optio
                 }
 
                 var xf = transforms[i];
+
+                var instanceName = $"{goMaster.name}_{i}";
+
+                // If the old instance exists, we must destroy it to avoid a duplicate
```

Comment:
since this commented section is repeated twice, could we turn this into a small function instead?

Reply:
I think they're different enough to be left alone (one looks at the existing instance, and one at its child) and the resulting function would only be like 2 lines long. Personally I stick to rule of only extracting to a separate function if there are 3 repeats, but I guess that's a personal preference :)